jina/resources/executors._unique.yml

```diff
@@ -1,6 +1,9 @@
 !DocIDCache
 with:
   index_path: cache.tmp
+  # TODO Should we rename _unique to _unique_id ?
```

Comment:
No, it should be `_unique_content_hash` or so. Since the old `id` is what the current `content_hash` is.

Reply:
Yes, but now we will have caches on the two possible fields:

-  `id` (which is not a hash, but passed by the user or assigned randomly)
- `content_hash` (which was the old `id`).

So we will need two `_unique_something.yml`, right?

The old one did you use `content_hash`, but it was actually using the field `id`. So the question is, which perspective do we preserve? The caller (which only cares it's `id`), or internal (which knows the old `id` is the new `content_hash`)?

Reply:
What I understood this will be `_unique` and will ensure that the document is not reindexed and so. I think `unique_something` would expose too much complexity to the user. What this does is guarantee that no duplicated is indexed while keeping it in sync when deletions or updates happen

Reply:
But "duplicated" based on what field? `id` should be the default. Even though `id` now means the random id assigned by user (or us), and not `content_hash` (which is what `id` used to be/mean)